MS Teams integration end user updates (#7373)
mattermost/docs: source/collaborate/collaborate-within-connected-microsoft-teams.rst

.. image:: ../images/microsoft-teams-chat-notifications.png
   :alt: An example of a chat message notification.

Managing notification settings
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

You can enable or disable notification settings for the Microsoft Teams integration at any time in **Account Settings** > **Plugin Preferences**

.. image:: ../images/teams_plugin_notification_settings.png
   :alt: Manage notification settings for the Microsoft Teams integration in Account Settings > Plugin Preferences

.. tip::
You can run the following slash commands to manage your integration settings by typing the commands into the Mattermost message text box, and selecting **Send**:

- ``/msteams connect``: Connect your Mattermost account to Microsoft Teams account.
- ``/msteams disconnect``: Disconnect your Mattermost account from Microsoft Teams account.
- ``/msteams status``: Show your current connection status.
- ``/msteams notifications on|off``: Change your current notifications settings.

FAQ

> ^^^^

Reviewer: ```suggestion
-----
```

How does the integration determine when to send chat notifications?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 

> Chat notifications are sent real-time whenever you're not active in Microsoft Teams and receive a chat or group chat; however, Mattermost uses your online status in Teams to determine if a chat notification should be delivered. Mattermost delivers notifications if you appear "Away" or "Offline" in Microsoft Teams, so the default behaviour for when notifications will be delivered depends on the client you typically use to access Microsoft Teams:

Reviewer: ```suggestion
Chat notifications are sent in real-time whenever you're not active in Microsoft Teams and receive a chat or group chat; however, Mattermost uses your online status in Teams to determine if a chat notification should be delivered. Mattermost delivers notifications if you appear **Away** or **Offline** in Microsoft Teams, so the default behavior for when notifications will be delivered depends on the client you typically use to access Microsoft Teams:
```

> - Web browser: Mattermost will deliver notifications if you've not had activity in your Microsoft Teams browser tab for 5 minutes, or when the browser tab is closed.  

Reviewer: ```suggestion
- Web browser: Mattermost delivers notifications when you've not had activity in your Microsoft Teams browser tab for 5 minutes or more, or when the browser tab is closed.  
```

> - Desktop App: Mattermost will deliver notifications if you've not had activity at your computer for 5 minutes, or when the Microsoft Teams desktop app is closed.

Reviewer: ```suggestion
- Desktop app: Mattermost delivers notifications when you've not had activity at your computer for 5 minutes or more, or when the Microsoft Teams desktop app is closed.
```

> In order to avoid double notifications, Mattermost will not deliver chat notifications if you appear as "Available", "Busy", "Do not disturb", or "Be right back" in Microsoft Teams.

Reviewer: ```suggestion
In order to avoid double notifications, Mattermost won't deliver chat notifications when your :ref:`Mattermost availability <preferences/set-your-status-availability:set-your availability>` is set to **Available**, **Busy**, **Do not disturb**, or **Be right back** in Microsoft Teams.
```
